webpack.config.js

```diff
@@ -0,0 +1,37 @@
+module.exports = (env, argv) => {
+    const path = require('path');
+    const webpack = require('webpack');
+
+    return {
+        entry: './src/index.ts',
+        output: {
+            path: path.resolve(__dirname, 'dist'),
+            filename: 'bundle.js'
+        },
+        resolve: {
+            modules: [path.join(__dirname, 'src'), 'node_modules'],
+            extensions: ['.js', '.ts', '.tsx']
+        },
+        module: {
+            rules: [
+                {
+                    test: /\.ts$/,
+                    use: [
+                        {
+                            loader: 'babel-loader',
```

Comment:
нам точно нужен сейчас babel? может пока выпилить его?

Reply:
А какой смысл его убирать на пару коммитов?
```suggestion
                            loader: 'babel-loader',
```